settings/common.py

```diff
@@ -483,5 +487,12 @@ def get_release():
 
 SITE_INFO = {
     'RELEASE_VERSION': RELEASE_VERSION,
-    'IS_RAVEN_INSTALLED': RAVEN_CONFIG['dsn'] is not ''
+    'IS_RAVEN_INSTALLED': RAVEN_CONFIG['dsn'] != ''
 }
+
+# SOCIAL MEDIA CONFIGURATION
+# -----------------------------------------------------------------------------
+MAX_POSTS_AT_ONCE = 5
```

Comment:
What is this used for?

Reply:
I tried to limit the number of posts that are posted at once. Like if there are 20 posts to be posted now, then all 20 will go out together. I thought that this might be intimidating, to see 20 posts all together come up in your feed, all saying similar stuff.

Reply:
That is why `scheduled_time` is there. If the admins have approved it, then it should go together if they're at the same time.

No need to restrict like this.

Reply:
I've added a flag, (which has False as default) in settings as discussed.

Reply:
Even after the flag, I should be able to configure this number. This should be in settings, picked from env. What do you think?

Reply:
@storymode7 Can you please reply here.